express-delete/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, () => {});
+
+app.get('/api/grades', (req, res) => {
+  const gradeArray = [];
+  for (const property in grades) {
+    gradeArray.push(grades[property]);
+  }
+  res.json(gradeArray);
+});
+
+app.delete('/api/grades/:id', (req, res) => {
+  delete grades[req.params.id];
+  res.sendStatus(204);
+});
```

Comment:
It'd be a good idea to respond with a `404 Not Found` if the client tries to delete a grade that doesn't exist.